app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    components = command.split()
+    if len(components) != 3 or components[0] != "cp":
```

Comment:
no need to use index here, u can use split method to unpack splitted string to varaiables, like this
`var1, var2, var3 = ur_string.split()`